test/blackbox-tests/test-cases/pkg/toolchain-variables.t

```diff
@@ -0,0 +1,76 @@
+Test the installation of toolchains package by building and installing
+a mock compiler package using dune's toolchain mechanism.
+
+  $ . ./helpers.sh
+  $ make_lockdir
+
+We create a fake compiler by creating a configure file.
+  $ mkdir fake-compiler
+  $ cat > fake-compiler/configure << 'EOF'
+  > #!/bin/sh
+  > PREFIX=$1
+  > echo $PREFIX > prefix.txt
+  > EOF
+  $ chmod a+x fake-compiler/configure
+
+We add a shell script to be installed as a fake compiler
+  $ mkdir -p fake-compiler/target/share/bin
+  $ cat > fake-compiler/target/share/bin/ocamlc << EOF
+  > #!/bin/sh
+  > echo "Hello from fake ocamlc!"
+  > EOF
+  $ chmod a+x fake-compiler/target/share/bin/ocamlc
+
+We make sure the installed script is installing the script at the correct
+location
+  $ cat > fake-compiler/Makefile << 'EOF'
```

Comment:
Do you really need make for this test? Would be better to just use a shell script.

Reply:
I've replaced it with a shell script now.